official/resnet/resnet_run_loop.py

```diff
@@ -434,31 +434,31 @@ def input_fn_eval():
             flags_obj.batch_size, flags_core.get_num_gpus(flags_obj)),
         num_epochs=1)
 
-  total_training_cycle = (flags_obj.train_epochs //
-                          flags_obj.epochs_between_evals)
-  for cycle_index in range(total_training_cycle):
+  for cycle_index in range(flags_obj.train_epochs):
     tf.logging.info('Starting a training cycle: %d/%d',
-                    cycle_index, total_training_cycle)
+                    cycle_index, flags_obj.train_epochs)
 
     classifier.train(input_fn=input_fn_train, hooks=train_hooks,
                      max_steps=flags_obj.max_train_steps)
 
-    tf.logging.info('Starting to evaluate.')
+    if cycle_index == (flags_obj.train_epochs //
```

Comment:
This line should be 
```python
if (cycle_index % flags_obj.epochs_between_evals) == 0:
```
otherwise the condition will only be satisfied once for the case `cycle_index == flags_obj.train_epochs // flags_obj.epochs_between_evals`.